src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -24,10 +28,10 @@
       text: az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p VerySecret --tenant contoso.onmicrosoft.com
     - name: Log in with a service principal using client certificate.
       text: az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p ~/mycertfile.pem --tenant contoso.onmicrosoft.com
-    - name: Log in using a VM's system-assigned managed identity.
+    - name: Log in with a system-assigned managed identity.
       text: az login --identity
-    - name: Log in using a VM's user-assigned managed identity. Client or object ids of the service identity also work.
-      text: az login --identity -u /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
+    - name: Log in with a user-assigned managed identity. You must specify the client ID, object ID or resource ID of the user-assigned managed identity with `--username`.
+      text: az login --identity --username /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
```

Comment:
According to Azure SDK, most app developers will prefer to specify a clientID here (a GUID). ResourceID is very exotic scenario, for example when you automate the provisioning of some hosts.

Reply:
Good catch! Changed as suggested.

Reply:
I also agree with Bogdan's suggestion on adding an example of specifying an ID, which you did.

I think it is a bit weird to provide a user-assigned managed identity (UAMI)'s GUID via a parameter named `username`. The word "username" would intuitively give me an impression that it would expect something like "JohnDoe" or "JohnDoe@contoso.com". That being said, if using `--username` to supply a UAMI's client_id is an established usage pattern, I would understand that it is too late to change it now.

Just a FYI. This PR looks good to me overall.

Reply:
> That being said, if using `--username` to supply a UAMI's client_id is an established usage pattern, I would understand that it is too late to change it now.

Exactly. It has been used since 2017 (https://github.com/Azure/azure-cli/pull/5140).